Description: glibc-2.16 defines TIME_UTC, which clashes with boost::TIME_UTC
Author: Dmitrijs Ledkovs <dmitrij.ledkov@ubuntu.com>
Last-Update: 2012-10-27

Index: b/boost/thread/xtime.hpp
===================================================================
--- a/boost/thread/xtime.hpp
+++ b/boost/thread/xtime.hpp
@@ -5,6 +5,10 @@
 //  Distributed under the Boost Software License, Version 1.0. (See accompanying 
 //  file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
 
+#ifdef TIME_UTC
+#undef TIME_UTC
+#endif
+
 #ifndef BOOST_XTIME_WEK070601_HPP
 #define BOOST_XTIME_WEK070601_HPP
 
